src/cpp/session/modules/build/SessionBuildErrors.cpp

```diff
@@ -293,15 +293,17 @@ std::vector<module_context::SourceMarker> parseTestThatErrors(
                   "\\s+"           // separating space
                   "\\("            // opening paren
                   kAnsiUrlRegex    // opening hyperlink
+                  kAnsiEscapeRegex // color
                   "([^:\\n]+)"     // file name           (2)
                   ":"              // colon separator
                   "([0-9]+)"       // file line           (3)
-                  ":"              // colon separator
-                  "([0-9]+)"       // file column         (4)
+                  "((?::)[0-9]+)?" // optional column     (4)
```

Comment:
Just to confirm, does this support the format used by old versions of (testthat / cli)? (Since some users might be running with older versions of R and so have whatever versions of testthat / cli were current at the time the new R release was published)

Reply:
I'll check that versions that emit the column number still work.

Reply:
Tested with `cli` 2.1.0 and `testthat` 3.0.1: 

The combo does emit the column number:

<img width="463" alt="image" src="https://user-images.githubusercontent.com/2625526/207549889-27e56e99-072a-4db3-b7cc-c031bb98d644.png">

and populates the table: 

<img width="346" alt="image" src="https://user-images.githubusercontent.com/2625526/207550050-2306e541-27df-429f-92d8-6856a172ead5.png">